static/app/components/events/traceEventDataSection.tsx

```diff
@@ -196,34 +197,19 @@ export function TraceEventDataSection({
                 title={t('Only full version available')}
                 disabled={hasAppOnlyFrames}
               >
-                <ButtonBar active={state.fullStackTrace ? 'full' : 'relevant'} merged>
-                  <Button
-                    size="xs"
-                    barId="relevant"
-                    onClick={() =>
-                      setState({
-                        ...state,
-                        fullStackTrace: false,
-                      })
-                    }
-                    disabled={!hasAppOnlyFrames}
-                  >
+                <SegmentedControl
+                  size="xs"
+                  aria-label={t('Include Frames')}
+                  value={state.fullStackTrace ? 'full' : 'relevant'}
+                  onChange={val => setState({...state, fullStackTrace: val === 'full'})}
+                >
+                  <SegmentedControl.Item key="relevant" disabled={!hasAppOnlyFrames}>
```

Comment:
When `hasAppOnlyFrames` is false:
<img width="221" alt="Screenshot 2023-01-31 at 10 11 41 AM" src="https://user-images.githubusercontent.com/44172267/215846904-17b6f838-42f4-4970-bb1f-f8f9728b9412.png">
<img width="221" alt="Screenshot 2023-01-31 at 10 12 12 AM" src="https://user-images.githubusercontent.com/44172267/215847008-56a032a1-63b8-4910-aed2-5acd2b54246f.png">